__tests__/app.test.js

```diff
@@ -44,6 +41,35 @@ describe("GET /api/topics", () => {
   });
 });
 
+describe("GET /api/articles/:article_id", () => {
+  test("200: returns and array with the selected article.", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then(({ body }) => {
+        expect(Object.keys(body.article)).toHaveLength(8);
+        expect(body.article).toMatchObject({
+          article_id: 1,
```

Comment:
Nice! Well done for making sure the test is still flexible but that we definitely have received back the correct article regardless of what the article info is!